src/control/conf.h

```diff
@@ -250,7 +250,6 @@ static inline void dt_conf_init(dt_conf_t *cf, const char *filename, GSList *ove
     }
     fclose(f);
   }
-  if(defaults) dt_configure_defaults();
```

Comment:
If you leave this in you shouldn't have the problem of figuring out if it's a new configuration or an old one lacking the setting.

Reply:
OK, makes sense. Let me restructure